src/main/java/org/concordion/api/CommandCall.java

```diff
@@ -90,4 +90,18 @@ public String getParameter(String camelCaseParameterName, String spinalCaseParam
         }
         return null;
     }
+
+    public boolean shouldExecuteWithNonExampleChildren() {
+        return this.command.alwaysExecuteEvenIfNoNonExampleChildren();

```

Comment:
For consistency with the method below, this should use `getCommand()` rather than `command`